src/pages/fiscal-years/forms/FiscalYearForm.js

```diff
@@ -0,0 +1,187 @@
+import { Grid } from '@mui/material'
+import { useContext, useEffect } from 'react'
+import * as yup from 'yup'
+import FormShell from 'src/components/Shared/FormShell'
+import toast from 'react-hot-toast'
+import { RequestsContext } from 'src/providers/RequestsContext'
+import { useInvalidate } from 'src/hooks/resource'
+import { ResourceIds } from 'src/resources/ResourceIds'
+import CustomTextField from 'src/components/Inputs/CustomTextField'
+import ResourceComboBox from 'src/components/Shared/ResourceComboBox'
+import { SystemRepository } from 'src/repositories/SystemRepository'
+import { validateNumberField } from 'src/lib/numberField-helper'
+import { VertLayout } from 'src/components/Shared/Layouts/VertLayout'
+import { Grow } from 'src/components/Shared/Layouts/Grow'
+import { useForm } from 'src/hooks/form'
+import CustomDatePicker from 'src/components/Inputs/CustomDatePicker'
+import { DataSets } from 'src/resources/DataSets'
+import { formatDateFromApi } from 'src/lib/date-helper'
+import { ControlContext } from 'src/providers/ControlContext'
+
+export default function FiscalYearForm({ labels, maxAccess, setStore, store }) {
+  const { getRequest, postRequest } = useContext(RequestsContext)
+  const { platformLabels } = useContext(ControlContext)
+  const { recordId } = store
+
+  const invalidate = useInvalidate({
+    endpointId: SystemRepository.FiscalYears.page
+  })
+
+  const { formik } = useForm({
+    initialValues: {
+      recordId: null,
```

Comment:
do we have recordId here?, in table u're passing rowId=fiscalYear

Reply:
needed to be passed to infobutton 
i am placing fiscal year value in it